rasa/core/policies/ted_policy.py

```diff
@@ -17,7 +17,15 @@
     MaxHistoryTrackerFeaturizer,
 )
 from rasa.core.featurizers.single_state_featurizer import SingleStateFeaturizer
-from rasa.shared.nlu.constants import ACTION_TEXT, ACTION_NAME, INTENT, TEXT, ENTITIES
+from rasa.shared.nlu.constants import (
+    ACTION_TEXT,
+    ACTION_NAME,
+    INTENT,
+    TEXT,
+    ENTITIES,
+    VALID_FEATURE_TYPES,
+    FEATURE_TYPE_SENTENCE,
```

Comment:
what is the difference to `SENTENCE` from `rasa.utils.tensorflow.constants`?

Reply:
basically the naming, would make it explicit and use the actual constant for `feature_type` when we are dealing with the feature type. just in case we change it.

Reply:
then, shall we rename constants in `rasa.utils.tensorflow.constants`?

Reply:
I'd stick to the shorter names, but I don't have strong opinion here